clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/names/GoogleLabeledResourceNamerSpec.groovy

```diff
@@ -50,24 +50,24 @@ class GoogleLabeledResourceNamerSpec extends Specification {
-    "app-cluster"                                             | [(CLUSTER): "myCluster"]                  || "app"        | "myCluster"                                          | "cluster"         | null           | null
-    "app-v042"                                                | [(SEQUENCE): "13"]                        || "app"        | "app"                                                | null              | null           | 13
-    "app-v042"                                                | [(DETAIL): "myDetail"]                    || "app"        | "app--myDetail"                                      | null              | "myDetail"     | 42
-    "awesomeapp--my-detail"                                   | null                                      || "awesomeapp" | "awesomeapp--my-detail"                              | null              | "my-detail"    | null
-    "awesomeapp--my-detail"                                   | getAllMonikerLabels(true)                 || "myApp"      | "myCluster"                                          | "myStack"         | "myDetail"     | 13
-    "awesomeapp--my-detail"                                   | getAllMonikerLabels(false)                || "myApp"      | "myApp-myStack-myDetail"                             | "myStack"         | "myDetail"     | 13
+    name                                                      | labels                                || expectedApp  | expectedCluster                                      | expectedStack     | expectedDetail | expectedSequence
+    "cass-nccpintegration-random-junk-d0prod-z0useast1a-v003" | null                                  || "cass"       | "cass-nccpintegration-random-junk-d0prod-z0useast1a" | "nccpintegration" | "random-junk" | 3
+    "cass-nccpintegration-random-junk-d0prod-z0useast1a-v003" | [:]                                   || "cass"       | "cass-nccpintegration-random-junk-d0prod-z0useast1a" | "nccpintegration" | "random-junk" | 3
+    "cass-nccpintegration-random-junk-d0prod-z0useast1a-v003" | [(APP): "myApp"]                      || "myApp"      | "cass-nccpintegration-random-junk-d0prod-z0useast1a" | "nccpintegration" | "random-junk" | 3
+    "cass-nccpintegration-random-junk-v003"                   | [(CLUSTER): "myCluster"]              || "cass"       | "myCluster"                                          | "nccpintegration" | "random-junk" | 3
+    "cass-nccpintegration-random-junk-v003"                   | [(STACK): "myStack"]                  || "cass"       | "cass-myStack"                                       | "myStack"         | "random-junk" | 3
+    "cass-nccpintegration-random-junk-v003"                   | [(STACK): "myStack", (DETAIL): ""]    || "cass"       | "cass-myStack"                                       | "myStack"         | ""            | 3
```

Comment:
What is the particular use tested in this case? Is it OK to have empty string detail -> empty string expected detail @jervi ?

Reply:
See my comment above

Reply:
But I guess it's OK. The result will be the same, I think.

Reply:
:+1: